rust/tools/authorization-logic/BUILD

```diff
@@ -34,7 +34,7 @@ java_binary(
 # Generate the parsing-related files using antlr4rust
 genrule(
     name = "antlr_gen_srcs",
-    srcs = ["src/parsing/AuthLogic.g4"],
+    srcs = ["//src/frontends/auth_logic:AuthLogic.g4"],
```

Comment:
Let us put this under `src/policy/auth_logic` unless there is strong objection from others.